tokens/sys/color/color.json

```diff
@@ -0,0 +1,582 @@
+          "value": "{palette.cinnamon.100}",
+          "type": "color",
+          "description": "Disabled error background"
+        },
+        "strong": {
+          "value": "{palette.cinnamon.600}",
+          "type": "color",
+          "description": "Strong error background"
+        },
+        "stronger": {
+          "value": "#80160E",
+          "type": "color",
+          "description": "Stronger error background"
+        }
+      }
+    },
+    "text": {
+      "default": {
+        "value": "{palette.black-pepper.300}",
+        "type": "color",
+        "description": "Default text color"
+      },
+      "disabled": {
+        "value": "{palette.licorice.100}",
+        "type": "color",
+        "description": "Disabled text color"
+      },
+      "hint": {
+        "value": "{palette.licorice.300}",
+        "type": "color",
+        "description": "Hint text color"
+      },
+      "strong": {
+        "value": "{palette.black-pepper.400}",
+        "type": "color",
+        "description": "Heading text color"
+      },
+      "stronger": {
+        "value": "{palette.black-pepper.500}",
+        "type": "color",
+        "description": "Heading hover text color"
+      },
+      "inverse": {
+        "value": "{palette.french-vanilla.100}",
+        "type": "color",
+        "description": "Inverse text color"
+      },
+      "critical": {
+        "default": {
+          "value": "{palette.cinnamon.500}",
+          "type": "color",
+          "description": "Error text"
+        }
+      },
+      "primary": {
+        "default": {
+          "value": "{palette.blueberry.400}",
+          "type": "color",
+          "description": "Branded text"
+        },
+        "strong": {
+          "value": "{palette.blueberry.500}",
+          "type": "color",
+          "description": "Branded hover text"
+        }
+      },
+      "caution": {
+        "default": {
+          "value": "{color.text.strong}",
+          "type": "color",
+          "description": "Warning text"
+        },
+        "strong": {
+          "value": "{color.text.stronger}",
+          "type": "color",
+          "description": "Strong warning text"
+        }
+      }
+    },
+    "icon": {
+      "default": {
+        "value": "{palette.licorice.200}",
+        "type": "color",
+        "description": "Default icon color"
+      },
+      "soft": {
+        "value": "{palette.licorice.100}",
+        "type": "color",
+        "description": "Disabled icon color"
+      },
+      "strong": {
+        "value": "{palette.licorice.500}",
+        "type": "color",
+        "description": "Hover icon color"
+      },
+      "inverse": {
+        "value": "{palette.french-vanilla.100}",
+        "type": "color",
+        "description": "Inverse icon color"
+      },
+      "primary": {
+        "default": {
+          "value": "{palette.blueberry.400}",
+          "type": "color",
+          "description": "Brand icon color"
+        }
```

Comment:
Question: Do we also need a `strong` here like we have for `color.text.primary`?

![button hover gif](http://g.recordit.co/6eBp3iEtIk.gif)